contrib/systemd/electrumx.conf

```diff
@@ -7,5 +7,7 @@ DAEMON_URL = http://username:password@hostname:port/
 
 # COIN = Bitcoin
 
+SERVICES = "tcp://0.0.0.0:50001,rpc://0.0.0.0:8000"
```

Comment:
First of all, why do you define this at all here? The main script has reasonable defaults if SERVICES is unset:
https://github.com/spesmilo/electrumx/blob/7e53936f34d2f1c979c64f91d15d6745fb8103c2/electrumx/server/env.py#L163-L165

Secondly, it is a horrible default to expose the local rpc server (admin controls) to the internet (0.0.0.0).

Not sure what the point of this PR is. I guess it's linked to your other PR (https://github.com/spesmilo/electrumx/pull/174) and you simply want to explicitly set the local rpc port to 8000 here... ?

Reply:
Thanks for clarification.